gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc

```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
-. **xref:flex-gateway-replicas.adoc[Manage Flex Replicas]**: By default Flex Gateway runs a single replica, but for production you will need more than one Flex Replica. Your replica should run in the same mode you registered your Flex Gateway.
-. **Manage an API with Flex Gateway:** Use your Flex Gateway to manage API instances and apply xref:policies::policies-included-overview.adoc[included], xref:policies::policies-automated-overview.adoc[automated], and xref:policies::policies-custom-overview.adoc[custom] policies.
-* xref:flex-conn-manage.adoc[Manage an API in Connected Mode]
-* xref:flex-local-manage.adoc[Manage an API in Local Mode]
\ No newline at end of file
+The goal of this tutorial is to quickly get started with Anypoint Flex Gateway. Using Docker, the tutorial describes how to install, register, and run Flex Gateway in Connected Mode. Additionally, the tutorial describes how to publish an API that is secured with a basic authentication policy.
+
+== Before You Begin
+
+Before getting started with Flex Gateway, ensure that you have:
+
+* A username and password for your Anypoint Platform organization. If you don’t have Anypoint Platform yet, https://anypoint.mulesoft.com/login/signup[create a Trial organization^] on Anypoint Platform.
+
+* A machine with an x86 processor. M1 and M2 processors are not supported.
+
+* A Docker installation. See https://www.docker.com/products/docker-desktop/[Docker Desktop^] for an example.
+
+== Install Flex Gateway
+
+To install the Flex Gateway Docker image, run the following command in a terminal:
+
+[source,ssh]
+----
+docker pull mulesoft/flex-gateway:latest
+----
+
+[NOTE]
+Use `sudo` if you encounter file permission issues when running this command.
+
+== Register Flex Gateway
+
+. Create a new directory by running the following command:
++
+[source,ssh]
+----
+mkdir flex-config
+----
+
+. Navigate to the new directory by running the following command:
++
+[source,ssh]
+----
+cd flex-config
+----
+
+. Log in to https://anypoint.mulesoft.com/login/[Anypoint Platform^] with your Anypoint username and password.
+. Select *Runtime Manager*.
+. Click on the *Flex Gateways* tab in the side navigation panel.
+. Click *Add Gateway*.
+. Select *Docker*.
+. Copy the code block in the *Register your gateway* section. The code block will contain your unique organization ID and token.
++
+The following screenshot of the *Add a Flex Gateway* page highlights the code block that should be copied.
++
+image::gateway-quickstart-add-gateway.png["Screenshot of 'Add a Flex Gateway'",width=90%]
+
+. Run the copied code block in your terminal window, making sure to replace `<gateway-name>` with a name, such as `my-gw`.
++
+[NOTE]
+Use `sudo` if you encounter file permission issues when running this command.
+
+. Select *Flex Gateways* in the side navigation panel.
++
+Your registered Flex Gateway now appears in the list of gateways.
+
+== Run Flex Gateway
+
+To run Flex Gateway, run the following command:
```

Comment:
```suggestion
To run Flex Gateway, enter the following in a terminal window:
```

Reply:
Hmm, I think I see what you're getting at. But they're not just entering text; they're also running the command.

Maybe:
"To run Flex Gateway, execute the following in a terminal window"?

Reply:
"Run" it is! :)